src/ucp/wireup/address.c

```diff
@@ -54,7 +54,10 @@ typedef struct {
 
 typedef struct {
     float            overhead;
-    float            bandwidth;
+    struct {
+        float dedicated;
+        float shared;
+    }                bandwidth;
```

Comment:
can we try to compress it to avoid increasing worker address size?
for example, most interfaces will have either dedicated or shared but not both
so add some flag / use sign bit to indicate if we have one float number or 2 float numbers
(could add this optimization in another PR)